src/modules/app/containers/App.js

```diff
@@ -11,6 +11,13 @@ import { ThemeProvider } from 'styled-components'
 import createRouteConfig from '../createRouteConfig'
 import theme from '../constants/theme'
 import I18nProvider from './I18nProvider'
+import { MuiThemeProvider } from '@material-ui/core'
+import muiTheme from '../constants/muiTheme'
+import { MuiPickersUtilsProvider } from 'material-ui-pickers'
+import MomentUtils from 'material-ui-pickers/utils/moment-utils'
+// We need to import the german locales for moment: https://material-ui-pickers.firebaseapp.com/localization/moment
+import 'moment/locale/de'
```

Comment:
What about other locales? Does it use them automatically?

Reply:
We don't have other locales, so no I don't think so.